tools/apksize/apksize.gradle

```diff
@@ -50,6 +51,36 @@ task uploadMeasurements(type: UploadMeasurementsTask) {
     uploader = "https://storage.googleapis.com/firebase-engprod-metrics/upload_tool.jar"
 }
 
+// TODO(yifany@): Move below 2 tasks out of apksize.gradle.
+task generateCoverageMeasurements(type: GenerateCoveragePercentsTask, group: 'Coverage') {
+
+    // TODO(yifany@): Find a more concise way to express dependencies.
+    dependsOn ':firebase-common:checkCoverage'
+    dependsOn ':firebase-database:checkCoverage'
+    dependsOn ':firebase-database-collection:checkCoverage'
+    dependsOn ':firebase-firestore:checkCoverage'
+    dependsOn ':firebase-functions:checkCoverage'
+    dependsOn ':firebase-inappmessaging-display:checkCoverage'
+    dependsOn ':firebase-storage:checkCoverage'
+
+    packages = ["firebase-common",
```

Comment:
Build this array using the project objects.

Reply:
Done.